test/e2e/API_Performance_Test/api_performance_tests.go

```diff
@@ -377,16 +377,24 @@ var _ = ginkgo.Describe("localstorage volume test ", ginkgo.Label("api"), func()
 
 		})
 
-		ginkgo.It("print test results", func() {
+		ginkgo.It("check results", func() {
 
-			result1, err := utils.RunInLinux(" grep -c '\"Failure Count\"' hwamei_stats.json ")
-			result2, err := utils.RunInLinux(" grep -c '\"Failure Count\": \"0\"'  hwamei_stats.json")
+			result1, err := utils.RunInLinux(" grep -c '\"Failure Count\"' hwamei_test_report* ")
+			result2, err := utils.RunInLinux(" grep -c '\"Failure Count\": \"0\"'  hwamei_test_report*")
 			if err != nil {
 				logrus.Error(err)
 			}
 			gomega.Expect(result1).To(gomega.Equal(result2))
 
 		})
+		ginkgo.It("Upload Report", func() {
```

Comment:
is this report can be accessed from public? if can, comment the report address here may be better

Reply:
We will consider updating this to the public website later.